src/mame/brother/lw30.cpp

```diff
@@ -0,0 +1,1105 @@
+// license:BSD-3-Clause
+// copyright-holders:Bartman/Abyss
+
+#include "emu.h"
+#include "emupal.h"
+#include "screen.h"
+#include "speaker.h"
+#include "machine/timer.h"
+#include "cpu/z180/z180.h"
+#include "imagedev/floppy.h"
+#include "util/utf8.h"
+
+#include "sound/beep.h"
+#include "video/mc6845.h"
+
+#define LOG_FLOPPY (1U << 1)
+#define LOGFLOPPY(...) LOGMASKED(LOG_FLOPPY, __VA_ARGS__)
+
+//#define VERBOSE (LOG_GENERAL | LOG_FLOPPY)
+//#define VERBOSE (LOG_GENERAL)
+#include "logmacro.h"
+
+// if updating project, c:\users\chuck\downloads\msys64\win32env.bat
```

Comment:
General usage notes are fine, but this all looks is pretty specific to your system,so there's no need to have them in the driver.